### DISCUSSION TITLE
Global search - custom fields of type 'selection' not appearing in results

### DISCUSSION THREAD

**Discussion by PurplePetrus** [netbox-community/netbox#13430] [Q&A]
Hi,

**some context;**

I'm trying out the custom fields in netbox. 
First of all, the outcome looks better than expected (mainly due to the grouping).
Configuring them is also no problem.

When i try to global search on them, the custom fields, it works for most of them.
Only the custom fields of the type 'selection' are not popping up in the results.

i like using the 'selection' type so i can prevent other users of typing in any fantasies.
But it's important i'm able to search on them as well. preferably using the global search on the top of the page.

Currently we can 'filter' them, but it's a hassle to use that when searching among different types at the same time.(devices, virtual machines,...)

**my question;**

Not being able to global search on the custom fields of the type 'selection', is this a bug or by design?
Will this be implemented in future versions?

**Comment by haristku** [comment]
I cannot "global search" the value of selection field too...

in Rack form, fields (Dimension Type & Width) are selection type, and I want to "global search" for example: "19 Inches" or "Wall-Mounted frame"

**Reply by PurplePetrus** [reply]
I've got Issue #13579 open for this now.
Apparently it's by design. 
I hope they accept the modification, it would be huge help.
